signup/js/signup.js

```diff
@@ -1,26 +1,60 @@
-const PWD_VIEWS = document.querySelectorAll('.pwd-view');
-const PWD_CHECK = document.querySelector('.pwd-confirm');
+import { moduleSign } from '/js/common.js';
 
-PWD_VIEWS.forEach((target) => {
+const INPUT_EMAIL = document.querySelector('#email');
+const INPUT_PASSWORD = document.querySelector('#password');
+const FORM = document.querySelector('#signup-form-box');
+const FORM_INPUT = document.querySelectorAll('#signup-form-box input');
+const PASSWORD_VIEWS = document.querySelectorAll('.pwd-view');
+const PASSWORD_CHECK = document.querySelector('.pwd-confirm');
```

Comment:
변수명과 클래스명의 표현을 맞추는게 좋을 것 같아요!
개인적으론 confirm이 더 명확해 보이네요 :)

```suggestion
const PASSWORD_CONFIRM = document.querySelector('.pwd-confirm');
```